src/pages/Cart/Cart.jsx

```diff
@@ -39,9 +39,9 @@ export const Cart = () => {
                                 </div>
                             </div>
                             <div className={`${styles.updateItems}`}>
-                                <button>-</button>
-                                <input type="text" value="1" />
-                                <button>+</button>
+                                <button onClick = { () => dispatch(decreaseItem({ _id, imgSrc, title, type, price, inStock, fastDelivery, onSale, discount, qty })) }>-</button>
```

Comment:
generally when we have to type a lot of values with " , "
Everything on a separate line makes a lot more sense and the cherry on the cake: its more readable as well.
```
return {
        id,
        fname,
        username,
        encodedToken,
        cartLength: Number(cartLength),
        wishlistLength: Number(wishlistLength),
      };